### PR TITLE
Use /var/lib/dcos/mesos for metrics module state path

### PR DESCRIPTION
**Pull request by spahl**
High level description including:
- fixed the state directory of the mesos volume to go to the standard path

# Issues

n/a

# Checklist

 - [ ] Included a test which will fail if code is reverted but test is not
 - [x] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

If this is a component/package update, include

 - [ ] Change Log from last: n/a
 - [ ] Test Results: n/a
 - [ ] Code Coverage: n/a

### CODE REVIEW

**Comment by alberts** [comment]
@spahl apparently this is about 1 kB of state that doesn't need to be persisted. I'd probably recommend that we put this under /run

**Comment by spahl** [comment]
ok let's move this to /run

File: packages/dcos-metrics/extra/metrics.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
             {"key": "output_collector_chunk_size_datapoints", "value": "100"},
             {"key": "output_collector_chunk_timeout_seconds", "value": "10"},
             {"key": "state_path_dir",
-             "value": "/var/run/mesos/isolators/com_mesosphere_MetricsIsolatorModule/"}
+             "value": "/var/lib/dcos/mesos/isolators/com_mesosphere_MetricsIsolatorModule/"}
```
Comment (spahl) [inline packages/dcos-metrics/extra/metrics.json]: this module creates it.

File: packages/dcos-metrics/extra/metrics.json [hunk 2]
```diff
@@ -16,7 +16,7 @@
             {"key": "output_collector_chunk_size_datapoints", "value": "100"},
             {"key": "output_collector_chunk_timeout_seconds", "value": "10"},
             {"key": "state_path_dir",
-             "value": "/var/run/mesos/isolators/com_mesosphere_MetricsIsolatorModule/"}
+             "value": "/run/dcos/mesos/isolators/com_mesosphere_MetricsIsolatorModule/"}
```
Comment (orsenthil) [inline packages/dcos-metrics/extra/metrics.json]: Question.

Don't have to create this path before using?
I see at-least metronome doing this ( https://github.com/dcos/dcos/blob/master/packages/metronome/build#L27)

**Pull request merged**